### PR TITLE
Build untrusted PR once when it is approved.

### PR DESCRIPTION
**Pull request by xiangpengzhao**
When an untrusted PR is approved, allow it to test once, as what is done for untrusted lgtm'd PR.
/cc @cjwagner
WDYT?

### CODE REVIEW

File: prow/plugins/trigger/pr.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func handlePR(c client, trustedOrg string, pr github.PullRequestEvent) error {
 		}
 	case github.PullRequestActionLabeled:
 		// When a PR is LGTMd, if it is untrusted then build it once.
```
Comment (kargakis) [inline prow/plugins/trigger/pr.go]: This comment needs to be updated

Comment (xiangpengzhao) [reply prow/plugins/trigger/pr.go]: Done. Thanks!

**Comment by cjwagner** [comment]
I like the idea of considering either of these as sufficient to start a round of testing, but there is a bit of problem. Neither the `lgtm` nor the `approved` label or command make a PR trusted, so if one occurs after the other testing will be triggered for both.
Since most repos require `lgtm` and `approved` labels, untrusted PRs would frequently trigger an unnecessary round of testing that supersedes the first round.
Consider an untrusted PR that has the been `lgtm`ed and passed tests. It still needs the `approved` label. If applying the `approved` label triggered another round of testing the PR would have to wait for the tests to pass again before merging.

Instead of triggering `buildAll` (basically the `/test all` command), the application of the `lgtm` or `approved` label should have the behavior of the `/retest` command and only retest jobs that have failed or are missing.

**Comment by xiangpengzhao** [comment]
@cjwagner sounds very reasonable. Thanks for clarification!
/close

**Pull request closed without merging**